packages/commerce-sdk-react/src/hooks/useMutation.ts

```diff
@@ -84,7 +91,17 @@ export const useCustomMutation = <TData = unknown, TError = unknown>(
             organizationId: globalConfig.organizationId,
             shortCode: globalConfig.shortCode
         },
-        proxy: globalConfig.proxy
+        proxy: globalConfig.proxy,
+        throwOnBadResponse: true
```

Comment:
This is required or else an HTTP 4xx will be considered a success by react query.

Reply:
is this a breaking change?

Reply:
I don't think so. We already throwOnBadResponse everywhere else (refer to the client config in the CommerceAPIProvider and Auth class). I think it was just a toggle we forgot to enable when support for custom APIs were added so this is a bugfix if anything.